openslides_backend/action/speaker/create_update_delete.py

```diff
@@ -1,9 +1,16 @@
 from ...models.models import Speaker
 from ..action_set import ActionSet
 from ..default_schema import DefaultSchema
+from ..generics import CreateAction, DeleteAction, UpdateAction
 from ..register import register_action_set
 
 
+class SpeakerDeleteAction(DeleteAction):
```

Comment:
This class seems unnecessary?

Reply:
At the moment we do not have internal actions so I need this class if I do not want to register it.

Reply:
Changed for other reasons.